packages/@aws-cdk/lambda-layer-kubectl/layer/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM public.ecr.aws/lambda/provided:latest
 # versions
 #
 
-ARG KUBECTL_VERSION=1.20.0
+ARG KUBECTL_VERSION=1.21.0
```

Comment:
Version 1.21 of `kubectl` is not compatible with versions of the server lower than 1.21. We haven't come up with a long term strategy for it yet. For now, please leave this version as it is.